### DISCUSSION TITLE
Run on STM32

### DISCUSSION THREAD

**Discussion by AndiArvy** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#423] [Q&A] [answered]
can this library run on stm32F103C8T6 or bluepill?

**Comment by mrcodetastic** [comment, marked as answer]
No. ESP32 only.

**Comment by board707** [comment]
> can this library run on stm32F103C8T6 or bluepill?

To control HUB75 panels on STM32 or RP2040 boards you can try [DMD_STM32 library](https://github.com/board707/DMD_STM32)
